### PR TITLE
job logs to grafana dashboard

### LINKED ISSUE #538: observability setup (monitoring and logging)
### What is the expected enhancement?

A simple experience of accessing logs and metrics on executed jobs

### PR DESCRIPTION
**Pull request by akihikokuroda**
### Summary
Fix #538 

The job execution logs are shown in the Grafana dashboard

### Details and comments

The Job execution logs in the ray head node are extracted by the fluent-bit sidecar container running in the ray head node to the sidecar container standard output.  It is pulled by the Promtail and pushed into Grafana dashboard (Job Logs) via Loki.

### CODE REVIEW

**Comment by psschwei** [comment]
The sidecar errored out on me:

```console
$ k logs kuberay-head-fj7gx ray-head-logs
Fluent Bit v1.9.6
* Copyright (C) 2015-2022 The Fluent Bit Authors
* Fluent Bit is a CNCF sub-project under the umbrella of Fluentd
* https://fluentbit.io

[2023/05/18 20:48:43] [ info] [fluent bit] version=1.9.6, commit=b30141ba90, pid=1
[2023/05/18 20:48:43] [ info] [storage] version=1.2.0, type=memory-only, sync=normal, checksum=disabled, max_chunks_up=128
[2023/05/18 20:48:43] [ info] [cmetrics] version=0.3.5
[2023/05/18 20:48:43] [error] [plugins/in_tail/tail_fs_inotify.c:304 errno=24] Too many open files
[2023/05/18 20:48:43] [error] Failed initialize input tail.0
[2023/05/18 20:48:43] [error] [lib] backend failed
```

**Comment by akihikokuroda** [comment]
@psschwei I can not make it happen.  What environment are you using?

File: infrastructure/helm/quantumserverless/templates/rayjoblogs.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          },
+          "id": 1,
+          "options": {
+            "dedupStrategy": "none",
+            "enableLogDetails": true,
+            "prettifyLogMessage": false,
+            "showCommonLabels": false,
+            "showLabels": false,
+            "showTime": false,
+            "sortOrder": "Descending",
```
Comment (psschwei) [inline infrastructure/helm/quantumserverless/templates/rayjoblogs.yaml:62]: nit: to make the logs easier to read
```suggestion
            "sortOrder": "Ascending",
```

**Review by psschwei — APPROVED** [review]

File: infrastructure/helm/quantumserverless/templates/rayjoblogs.yaml (hunk 1 above)
Comment (akihikokuroda) [reply infrastructure/helm/quantumserverless/templates/rayjoblogs.yaml:62]: I don't have any strong opinion but the "Decending" is default in Grafana logs.  People may like it better.

Comment (psschwei) [reply infrastructure/helm/quantumserverless/templates/rayjoblogs.yaml:62]: the default view I got was newest logs on top (i.e. append to beginning of file) which I found extremely counter-intuitive

Comment (akihikokuroda) [reply infrastructure/helm/quantumserverless/templates/rayjoblogs.yaml]: Here is one of the discussion.  https://github.com/grafana/grafana/issues/16555.

**Pull request merged**